api/src/main/resources/application.yaml

```diff
@@ -0,0 +1,59 @@
+#Keycloak/OAuth properties
+security:
+  oauth2:
+    resource:
+      jwt:
+        key-value: ${KEYCLOAK_PUBLIC_KEY}
+
+#DB Properties
+spring:
+  jmx:
+    enabled: false
+  mvc:
+    log-request-details: ${SPRING_SHOW_REQUEST_DETAILS}
+  jackson:
+    deserialization:
+      fail-on-unknown-properties: false
+
+#Logging properties
+logging:
+  file:
+    max-history: 1
+    clean-history-on-start: true
+    max-size: 20MB
+    name: /logs/app.log
+  #pattern:
+  #  file: %d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+  #  console: %d | [${HOSTNAME}] | %clr(%5p) | [%t]   | %logger{36}:%L | %msg%n
+  level:
+    ca:
+      bc:
+        gov:
+          educ:
+            api:
+              graduation: ${APP_LOG_LEVEL}
```

Comment:
The package name for log level needs to be corrected.  ( graduation -> distribution )

Reply:
Ok..I have made this change :)